mvpa2/datasets/mri.py

```diff
@@ -270,8 +283,18 @@ def fmri_dataset(samples, targets=None, chunks=None, mask=None,
             ds.fa[fattr] = ds.a.mapper.forward1(value)
 
     # store interesting props in the dataset
-    ds.a['imghdr'] = imghdr
-    ds.a['imgtype'] = imgtype
+    ds.a['imgaffine'] = img.get_affine()
```

Comment:
how about using get_best_affine() instead? According to the documentation, this "Select[s] best of available transforms"